CHANGELOG.md

```diff
@@ -651,6 +651,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Generate valid xcconfig when target name includes spaces.
```

Comment:
@segiddins I am not seeing what you mean with missing trailing two spaces...

I fixed the location of the changelog though for my fix...